client/state/notices/middleware.js

```diff
@@ -300,6 +301,26 @@ const onSiteDeleteFailure = ( { error } ) => {
 	return errorNotice( error.message );
 };
 
+const onBillingTransactionRequestFailure = ( { transactionId, error } ) => {
+	if ( 'invalid_receipt' === error.error ) {
+		return errorNotice(
+			translate( "Sorry, we couldn't find receipt #%s", { args: transactionId } ),
+			{
+				id: `transaction-fetch-${ transactionId }`,
```

Comment:
The `id` and `displayOnNextPage` are duplicated on the next `errorNotice` - would you like to set those up in a constant above and reuse them in both `errorNotice` instances?